mesh/mesh.cpp

```diff
@@ -4231,6 +4600,25 @@ void Mesh::EnsureNodes()
    }
 }
 
+void Mesh::SetVerticesFromNodes()
```

Comment:
There already is a very similar method one page below, is the new one necessary?

Reply:
No, it's not necessary. I removed it.